main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+	r := mux.NewRouter()
+	SetupRouter(r)
```

Comment:
I think that `SetupRouter` could easily be responsible for creating new router.

```suggestion
	r := SetupRouter()
```